spec/features/dashboard/advisers_edit_spec.rb

```diff
@@ -0,0 +1,113 @@
+RSpec.feature 'The dashboard firm edit page' do
+  include CheckboxGroupHelpers
+
+  let(:advisers_index_page) { Dashboard::AdvisersIndexPage.new }
+  let(:adviser_edit_page) { Dashboard::AdviserEditPage.new }
+  let(:original_postcode) { 'L1 2NH' }
+  let(:updated_postcode) { 'EH11 2AB' }
+
+  before do
+    5.times { |num| FactoryGirl.create :accreditation, order: num }
+    5.times { |num| FactoryGirl.create :qualification, order: num }
+    5.times { |num| FactoryGirl.create :professional_standing, order: num }
+    5.times { |num| FactoryGirl.create :professional_body, order: num }
```

Comment:
@tomas-stefano We've raised a PR for this in the other repo: https://github.com/moneyadviceservice/mas-rad_core/pull/70, do you mind taking a look?